src/core/public/keyboard_shortcut/keyboard_shortcut_service.ts

```diff
@@ -18,15 +18,20 @@ import { KeyboardShortcutSetup, KeyboardShortcutStart, ShortcutDefinition } from
 export class KeyboardShortcutService {
   private shortcutsMapByKey = new Map<string, ShortcutDefinition[]>();
   private namespacedIdToKeyLookup = new Map<string, string>();
+  private enabled = true;
 
```

Comment:
Can just make this the config in case in the future we need to add more configurations to this service